src/modules/Buttons.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable array-callback-return */
+import React from 'react';
+
+export default class Buttons extends React.Component {
+  constructor(props) {
```

Comment:
- [ ] As written in the project requirements, your project should have a directory called `components` and 
- The Calculator.js component should be in the components folder above.
- Ensure the `Calculator.js` component is class-based.